pricing.js

```diff
@@ -1,36 +1,41 @@
+function volLife(product, price, dollarsOff) {
```

Comment:
This function is determining the employer contribution and updating the price. Remember we should name functions what they do so the name `volLife` would be pretty misleading.